src/types/synchronization.d.ts

```diff
@@ -58,15 +60,24 @@ export interface SynchronizationState {
     done?: boolean;
 }
 
+interface NamedRef extends Ref {
+    name: string;
+}
```

Comment:
@tokland I believe I have seen this type of references somewhere in the API. I have pending to check if we already have a type in ``d2-api`` for this.